app/views/layouts/application.html.erb

```diff
@@ -2,10 +2,9 @@
 <html>
 <head>
   <title>RailsReactTutorial</title>
-  <%= env_stylesheet_link_tag 'application_prod', 'application_dev', media: 'all', 'data-turbolinks-track' => true  %>
-  <%= env_javascript_include_tag nil, 'http://localhost:3500/vendor-bundle.js' %>
-  <%= env_javascript_include_tag nil, 'http://localhost:3500/app-bundle.js' %>
-  <%= env_javascript_include_tag 'application_prod', 'application_dev', 'data-turbolinks-track' => true %>
+  <%= env_stylesheet_link_tag(static: 'application_static', hot: 'application_hot', options: { media: 'all', 'data-turbolinks-track' => true })  %>
+  <%= env_javascript_include_tag(hot: ['http://localhost:3500/vendor-bundle.js', 'http://localhost:3500/app-bundle.js']) %>
+  <%= env_javascript_include_tag(static: 'application_static', hot: 'application_hot', options: { 'data-turbolinks-track' => true }) %>
```

Comment:
throw a comment in on why this is 2 lines, b/c we have different options for application_static and application_hot.